### ISSUE TITLE
Use more semantic containers

### ISSUE THREAD

**Issue by gregrickaby** [WebDevStudios/wd_s#275]
Currently, we use `.wrap` to create a container. Then, devs can create "whatever they want" to create rows and columns.

Why not take a cue from Bootstrap and do: "container --> row --> column"

```html
<div class="container">
  <div class="row">
    <div class="col col-6">
      1 of 2
    </div>
    <div class="col col-6">
      1 of 2
    </div>
  </div>
  <div class="row">
    <div class="col col-4">
      1 of 3
    </div>
    <div class="col-4">
      1 of 3
    </div>
    <div class="col col-4">
      1 of 3
    </div>
  </div>
</div>
```

Related ticket: #269

**Comment by carrieforde** [comment]
I like this a lot, and would definitely like to bring current wd_s up to this standard. My only hesitation right now is that we've got a lot of `wrap`s, which is fine, but I'm wondering if they really suit our needs 100%.

For example, on client projects where we've used ACF with flexible content, we've been using a `container` to wrap the module which allows us to more easily use full width, using a `row` to wrap the content which allows us to limit the width of the inside content, and `col-x` for any column needs within the module.

In that case, I'm wondering if it makes sense to have `container` within a `container` as we'd end up with, or if we can maybe remove / reconsider some of the existing `wrap`s?
